hs_model_program/views.py

```diff
@@ -37,10 +37,10 @@ def get_model_metadata(request):
                 software_language=mpmeta.modelProgramLanguage,
                 operating_sys=mpmeta.modelOperatingSystem,
                 url = protocol+"://"+request.get_host()+"/resource/"+resource_id+"/",
-                modelEngine = mpmeta.modelEngine.split(';'),
-                modelSoftware=mpmeta.modelSoftware.split(';'),
-                modelDocumentation=mpmeta.modelDocumentation.split(';'),
-                modelReleaseNotes=mpmeta.modelReleaseNotes.split(';'),
+                modelEngine = mpmeta.get_engine_list(),
```

Comment:
This view is called via ajax in the javascript to populate the multi-select elements in the resource edit mode: https://github.com/hydroshare/hydroshare/blob/c75c8dda3fa3fe73fff85e8c4c7a91abc3afb0e5/hs_model_program/static/js/model_program.js#L51